### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by rijkvanzanten**

### CODE REVIEW

**Review by br41nslug — DISMISSED** [review]

File: app/src/lang/translations/zh-CN.yaml [hunk 1]
```diff
@@ -1226,6 +1226,7 @@ directus_collection:
   directus_access: 策略附件
   directus_activity: 所有事件操作日志
   directus_collections: 其他集合配置和元数据
+  directus_comments:
```
Comment (AlexGaillard) [inline app/src/lang/translations/zh-CN.yaml:1229]: `undefined` will throw error

File: app/src/lang/translations/es-MX.yaml [hunk 2]
```diff
@@ -191,7 +191,7 @@ logoutReason:
 public_label: Público
 public_description: Controla qué datos de la API están disponibles sin autenticar.
 public_role_info: >-
-  El rol público controla qué datos de la API están disponibles para usuarios no autenticados o usuarios sin rol. Las solicitudes con el rol público no pueden tener acceso de app o administradores. Si se da acceso de app o administrador a una política asignada al rol público, serán ignoradas.
+
```
Comment (AlexGaillard) [inline app/src/lang/translations/es-MX.yaml:194]: `undefined` will throw error

**Review by AlexGaillard — COMMENTED** [review]
Two missing values here causing the tests to fail. Not sure what the regular POA is here? Just remove the affected lines?

File: packages/components/vite.config.ts.timestamp-1733238145028-3a042f1e1d909.mjs
[the remark this answers is not in the document]
Comment (AlexGaillard) [reply packages/components/vite.config.ts.timestamp-1733238145028-3a042f1e1d909.mjs:1]: Yep I saw 👍. Removing now.

**Review by AlexGaillard — APPROVED** [review]
Should be fixed now

**Comment by paescuj** [comment]
@AlexGaillard FYI: Since Crowdin updates are one-way sync, we need to resolve such issues directly on Crowdin (i.e. delete empty translations there). Otherwise the same issue will appear again in the next Crowdin PR (see #24170). I've now fixed it.
The corresponding check added in #21792 was put in place since empty translation strings would prevent the Data Studio from working correctly.

**Comment by AlexGaillard** [comment]
Ah, thank you 👍 Good to know

**Pull request merged**